### PR TITLE
chore: test utils cleanup

### PR DESCRIPTION
**Pull request by galvez**
Refactor `test/utils` to better separate Nuxt-specific imports/exports and generally clean up these files' headers. Also removed a comment mentioning a `createSportsSelectionView` which doesn't appear to be related to Nuxt at all.

### CODE REVIEW

File: packages/common/src/index.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
+import * as _Utils from './utils'
+
 export { default as Modes } from './modes'
 export { default as NuxtConfig } from './nuxt.config'
 export { default as Options } from './options'
 export * from './utils'
+export const Utils = _Utils
```
Comment (clarkdo) [inline packages/common/src/index.js]: Now we export an object `Utils` includes all functions in `./utils` and also all functions on `index`, it looks kind of confusing, we'd better keep only one behaviour.

Comment (galvez) [reply packages/common/src/index.js]: @clarkdo gotcha, makes sense. I'll modify.

**Comment by galvez** [comment]
@clarkdo updated, tests passing :)

**Review by pi0 — DISMISSED** [review]
LGTM

**Review by pi0 — APPROVED** [review]

**Pull request merged**